packages/server/lib/server-base.ts

```diff
@@ -118,23 +117,25 @@ export abstract class ServerBase<TSocket extends SocketE2E | SocketCt> {
   protected _networkProxy?: NetworkProxy
   protected _netStubbingState?: NetStubbingState
   protected _httpsProxy?: httpsProxy
-
-  protected _remoteAuth: unknown
-  protected _remoteProps: unknown
-  protected _remoteOrigin: unknown
-  protected _remoteStrategy: unknown
-  protected _remoteVisitingUrl: unknown
-  protected _remoteDomainName: unknown
-  protected _remoteFileServer: unknown
+  protected _eventBus: EventEmitter
+  protected remoteStates: RemoteStates
```

Comment:
Any reason for the difference in naming convention here?

Reply:
@mschile ?

Reply:
Updated [6ee45ac](https://github.com/cypress-io/cypress/pull/21137/commits/6ee45ac8fb50fbec9e3d87c24be9bc9757386856)